CONTRIBUTING.rst

```diff
@@ -0,0 +1,184 @@
+============
+Contributing
+============
+
+Bugs
+====
+
+All bugs are tracked in `<https://bugzilla.mozilla.org/>`_.
+
+Write up a new bug:
+
+https://bugzilla.mozilla.org/enter_bug.cgi?product=Webtools&component=Pontoon
+
+
+Docker
+======
+
+Everything runs in a Docker container. Thus Pontoon requires fewer things to get
+started and you're guaranteed to have the same setup as everyone else and it
+solves some other problems, too.
+
+If you're not familiar with `Docker <https://docs.docker.com/>`_ and
+`docker-compose <https://docs.docker.com/compose/overview/>`_, it's worth
+reading up on.
+
+
+Python code conventions
+=======================
+
+Python code should follow PEP-8.
+
+Max line length is 100 characters.
+
+4-space indentation.
+
+To run the linter, do::
+
+  $ pylama pontoon
```

Comment:
`pylama` should be enough, all default directories should be specified in `pylama.ini`